config/initializers/assets.rb

```diff
@@ -0,0 +1 @@
+Rails.application.config.assets.precompile += %w( refinery/blog/backend.js refinery/wymeditor.js )
```

Comment:
This extension should not be aware of wymeditor in this initializer, you should remove `refinery/wymeditor.js`

Reply:
If you have to precompile `refinery/blog/backend` you should add it in https://github.com/refinery/refinerycms-blog/blob/master/lib/refinery/blog/engine.rb#L16-L20